src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemIndex.html.twig

```diff
@@ -78,17 +78,11 @@
               <div class="btn-group pull-left" role="group">
                 {% if showMediaLibraryMediaFolderDelete %}
                   {% if not mediaFolder.hasConnectedItems and not mediaFolder.hasChildrenWithConnectedItems %}
-                    <button type="button" class="btn btn-danger" id="media-folder-delete" data-toggle="modal" data-target="#confirmMediaFolderDelete">
-                      {{ macro.icon('trash-o') }}
-                      {{ 'lbl.MediaFolderDelete'|trans|ucfirst }}
-                    </button>
+                    {{ macro.buttonIcon('', 'trash-o', 'lbl.MediaFolderDelete'|trans|ucfirst, 'btn-danger', { "id":"media-folder-delete", "data-toggle":"modal", "data-target":"#confirmMediaFolderDelete", "type":"button" } ) }}
                   {% endif %}
                 {% endif %}
                 {% if showMediaLibraryMediaFolderEdit %}
-                  <button type="button" class="btn" id="media-folder-edit" data-toggle="modal" data-target="#editMediaFolderDialog">
-                    {{ macro.icon('pencil') }}
-                    {{ 'lbl.EditMediaFolder'|trans|ucfirst }}
-                  </button>
+                  {{ macro.buttonIcon('', 'pencil', 'lbl.EditMediaFolder'|trans|ucfirst, 'btn-default', { "id":"media-folder-edit", "data-toggle":"modal", "data-target":"#editMediaFolderDialog", "type":"button" } ) }}
```

Comment:
All id's are camelcased, so this one should be too.

Reply:
This id is not used at all so it should simply be removed.